install/install.go

```diff
@@ -0,0 +1,191 @@
+package install
+
+import (
+	"archive/zip"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+func valid(module string) bool {
+	return strings.HasPrefix(module, "github.com/")
+}
+
+func Install(module string) {
+
+	if !valid(module) {
+		fmt.Printf(`Error reading URL. Please use "github.com/USER/REPO" format to install`)
+		return
+	}
+
+	err := getZip(module)
+	if err != nil {
+		return
+	}
+
+	err = unzip(module)
+	if err != nil {
+		return
+	}
+
+	err = createAlias(module)
+	if err != nil {
```

Comment:
if a module with the same alias exists we print funny stuff:

```
~/projects/abs (pr-277 ✘)✹ ᐅ ./builds/abs-1.8.0-linux-amd64 get github.com/abs-lang/abs 
Install Success. You can use the module with "require('abs')"
~/projects/abs (pr-277 ✘)✹ ᐅ ./builds/abs-1.8.0-linux-amd64 get github.com/mingwho/abs 
This module could not be aliased because module of same name existsInstall Success. You can use the module with "require('abs')"
~/projects/abs (pr-277 ✘)✹ ᐅ cat ./alias
{
    "abs": "./vendor/github.com/github.com/abs-lang/abs-master",
}
```

I think it's ok to warn the user that the 2nd module cant be aliased but this is what I would do:

* let the user know a module exists, can't be aliased
* print the help message mentioning you have to use the full path to the module eg. `You can use the module with "require('github.com/x/y')"`
* add an entry in the alias file regardless -- this will allow the user to manually update the alias file if they want

Also my suggestion is to rename the alias file. Alias is a very common word so might collide with other stuff. My suggestion would be to use `packages.abs.json` but I'm open to suggestions :)

Reply:
Thanks for the note! This part is done 👍